Importupdate (#386)
EBIvariation/eva-opentargets: docs/manual-curation/step2-manual-curation.md

## Curation workflow
Curation should be done by subsequently applying filters to appropriate columns, then making decisions for the traits in the filtered selection.

* 1\. **Suggested exact EFO mappings.** These are terms we can use confidently as they match the term to map perfectly and already reside in EFO. Filter options as follows:
  * 1.1. Remove "Blank" from *Suggested exact mapping* column 
  * 1.2. Filter by colour: Fill colour #B7E1CD

>   * 1.3. Use these mappings and mark as DONE

Reviewer: ```suggestion
  * 1.3. Use these mappings by copy/pasting them directly and marking as DONE
```
(or something like that, just slightly more explicit)

> * 2\. **Suggested previous mappings.** All of these are the decisions that we made in the past, so we trust them unless they have been made obsolete. In this case there will be a suggested replacement, or we can find an entirely new mapping. Filter options as follows:

Reviewer 2: I would actually have a separate point where we should detail how to detect and resolve Obsolete terms.
This point should probably come before `Suggested exact EFO mappings`

  * 2.1. Remove "Blank" from *Suggested previous mapping* column

>   * 2.2. Filter by colour: Fill colour light red 3

Reviewer: ```suggestion
  * 2.2. Filter by colour: Fill colour `light red 3`
```
I think we should probably add screenshots for a few of these steps, but we can come back and do that another time (unless you're feeling ambitious).  For now I think adding the tick marks helps clarify that these are the exact menu options you have to look for.

>   * 2.3. Determine if *Suggested replacement mapping* is suitable, if not find a new term to use as mapping

Reviewer: ```suggestion
  * 2.3. Determine if *Suggested replacement mapping* is suitable, if not find a new term to use as mapping as in step 3.
```

  * 2.4. Remove Fill colour light red 3 filter and check all other *Suggested previous mapping* using the following ruleset:

>     * 2.4.1. **The previously assigned mapping is in EFO and is exact.** Mark as finished immediately. (It's extremely unlikely that a better mapping could be found).

Reviewer 2: I think this case would already be taken care of in point 1. but I guess it does not hurt to keep it here

>     * 2.4.2. **The previously assigned mapping is in EFO and IS NOT exact.** Review the mappings to see if a better (more accurate/specific, non-obsolete) mapping is available. Then mark as finished.

Reviewer 2: I would suggest here to search for more precise exact mapping following this recipes:
- Filter the `Suggested exact mapping` for text containing `MONDO_` or `HP_` and compare the label to see if the exact mapping are more precise. These can be imported if they are more precise.

    * 2.4.3. **The previously assigned mapping is not contained in EFO and is not exact.** We need to either find a mapping which is already in EFO, or import these terms into EFO.
    * 2.4.4. **The previously used mapping IS NOT contained in EFO and is exact.** These are good candidates to mark as finished and them import in EFO afterwards.

>     * 2.4.5. **The previously assigned mapping IS NOT contained in EFO and IS NOT exact.** Similarly to 2.4.3, attempt to find an acceptable EFO mapping; if not found, use any acceptable mapping (with subsequent import to EFO).

Reviewer 2: Does this actually occur ? Is there previously assigned mapping (presumably we assigned it because it was in EFO) that are not in EFO anymore ?
